content/en/platform/corda/4.11/enterprise/node/setup/corda-configuration-fields.md

```diff
@@ -267,9 +267,10 @@ Allows fine-grained controls of various features only available in the enterpris
     * The password for TLS TrustStore.
     * *Default:* not defined
 * `messagingServerConnectionConfiguration`
-  * Mode used when setting up the Artemis client. Supported modes are: DEFAULT (5 initial connect attempts, 5 reconnect attempts in case of failure, starting retry interval of 5 seconds with an exponential back-off multiplier of 1.5 for up to 3 minutes retry interval),
-  * FAIL_FAST (no initial attempts, no reconnect attempts), CONTINUOUS_RETRY (infinite initial and reconnect attempts, starting retry interval of 5 seconds with an exponential back-of multiplier of 1.5 up for up to 5 minutes retry interval).
-  * *Default:* DEFAULT
+  * Mode used when setting up the Artemis client. Supported modes are: 
+    * **DEFAULT:** 5 initial connect attempts, 5 reconnect attempts in case of failure, starting retry interval of 5 seconds with an exponential back-off multiplier of 1.5 for up to 3 minutes retry interval.
```

Comment:
```suggestion
    * **DEFAULT:** Five initial connect attempts, five reconnect attempts in case of failure, starting retry interval of five seconds with an exponential back-off multiplier of one and a half for up to three minutes retry interval.
```
I spell out numbers lower than 10, I know Colin did too but not sure how consistent it is throughout the docs.